Cargo.toml

```diff
@@ -78,7 +78,7 @@ sugar_path       = { version = "1.2.0", features = ["cached_current_dir"] }
 syn              = { version = "2.0.95" }
 tokio            = { version = "1.42.0", features = ["rt", "rt-multi-thread"] }
 # don't include debug & trace in release binary to save release binary size
-tracing            = { version = "0.1.41", features = ["max_level_trace", "release_max_level_info"] }
+tracing            = { version = "0.1.41", features = ["max_level_trace", "release_max_level_trace"] }
```

Comment:
support trace level trace in release binary cause it cause 0.4M increase before gzip which seems acceptable